privacy/gdpr/policy_test.go

```diff
@@ -72,3 +72,32 @@ func TestWrite(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateConsent(t *testing.T) {
+	testCases := []struct {
+		description string
+		consent     string
+		expectError bool
+	}{
+		{
+			description: "Invalid",
+			consent:     "<any invalid>",
+			expectError: true,
+		},
+		{
+			description: "Invalid",
+			consent:     "BONV8oqONXwgmADACHENAO7pqzAAppY",
+			expectError: false,
+		},
+	}
```

Comment:
I don't care to test any other scenarios, since I don't want to focus on retesting the go-gdpr library here.